WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -309,6 +305,46 @@ public void onClick(View v) {
         });
     }
 
+    private void showFeaturedImage(int postId, WPNetworkImageView imgFeatured) {
+        String imageUrl = mFeaturedImageUrls.get(postId);
+        if (imageUrl == null) {
+            imgFeatured.setVisibility(View.GONE);
+        } else if (imageUrl.startsWith("http")) {
+            String photonUrl =  ReaderUtils.getResizedImageUrl(imageUrl, mPhotonWidth, mPhotonHeight, mSite.isPrivate());
```

Comment:
Instead of `mSite.isPrivate()` should we use `SiteUtils.isPhotonCapable(mSite)`?
Otherwise self hosted sites can get redirected to Photon.